support/ci/shared.ps1

```diff
@@ -20,7 +20,7 @@ function Install-Rustup($Toolchain) {
     if (get-command -Name rustup.exe -ErrorAction SilentlyContinue) {
         Write-Host "rustup is currently installed"
         rustup set default-host x86_64-pc-windows-msvc
-        rustup default stable-x86_64-pc-windows-msvc
+        rustup default "$(Rust-Toolchain)-x86_64-pc-windows-msvc"
```

Comment:
you will want to keep this `stable` because there is no guarantee that `$(Rust-Toolchain)-x86_64-pc-windows-msvc` will exist until `Install-RustToolchain` runs. Making it `stable` will not interfere with our desires to execute with the specific toolchain we are at since all `cargo` commands specify that toolchain. During the docker work, I found that setting the default to stable prevented some weird errors when running `Install-RustToolchain` when there was no default set at all which seems to be the case in our docker images.

Reply:
That's very helpful; will fix.